src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
```

Comment:
font inherit, no need to use `*` for it.
Add it to `body` and all elements will have it.